java/code/src/com/redhat/rhn/domain/server/Server.java

```diff
@@ -2429,6 +2450,10 @@ boolean isDebian10() {
         return ServerConstants.DEBIAN.equals(getOs()) && getRelease().equals("10");
     }
 
+    boolean isRHEL() {
+        return ServerConstants.RHEL.equals(getOs());
```

Comment:
can't we re-use the condition `ServerConstants.REDHAT.equals(getOsFamily())` instead of creating a new variable to defined RedHat?